plugin/al/mayautils/AL/maya/utils/MenuBuilder.cpp

```diff
@@ -117,7 +117,7 @@ void MenuBuilder::Menu::generate(
                 nameWithNoSpaces[i] = '_';
         }
         os << "if(`menu -exists " << nameWithNoSpaces << "`) return;\n";
-        os << "menu -parent $gMainWindow -l \"" << m_name << "\" -aob 1 " << nameWithNoSpaces
+        os << "menu -tearOff true -parent $gMainWindow -l \"" << m_name << "\" -aob 1 " << nameWithNoSpaces
```

Comment:
@fabal this is a super minor change but I found it useful to have the USD menu tear-off-able when trying different menu items and options.